src/System.Private.ServiceModel/tools/scripts/StartWCFService.cmd

```diff
@@ -19,14 +21,12 @@ if NOT [%ERRORLEVEL%]==[0] (
     goto :done
   )
 
-
 REM Config Certs
 REM we need the direcotry to save the test.crl file. We are investigate a way to get rid of it
 REM
 md c:\wcftest
-call %~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator\CertificateGenerator.exe %*
+%~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator\CertificateGenerator.exe
 set __EXITCODE=%ERRORLEVEL%
-
 if NOT [%__EXITCODE%]==[0] (
     echo Warning: An error occurred while running certificate generator.
   )
```

Comment:
I will explain why "1" sometimes are successful code. But I cannot explain non-zero as there could be too many error conditions.